### PR TITLE
A few doc updates to address bugs

### LINKED ISSUE #927: Is there a list of system-packages for python 2 & 3?
I've been trying to figure out if matplotlib plot directives are supported (#896) on python3 and would prefer not to rely on trial'n error.

Is it possible to have an updated list of the system's packages for python 2 AND python-3?

### PR DESCRIPTION
**Pull request by ericholscher**
Lots of little updates that fix various doc bugs we had outstanding.

### CODE REVIEW

File: docs/api.rst [hunk 1]
```diff
@@ -4,6 +4,9 @@ Read the Docs Public API
 We have a limited public API that is available for you to get data out of the site. 
 This document covers only part of the API provided. We have plans to create a read/write API, so that you can easily automate interactions with your project.
 
+.. warning:: This API is out of date and not currently maintained.
+             We have a v2 API that is in better repair available at http://readthedocs.org/api/v2/.
```
Comment (humitos) [inline docs/api.rst]: _is in better repair available_, is that phrase is correct? I'm not sure to understand what it means.

File: docs/faq.rst [hunk 2]
```diff
@@ -151,11 +149,6 @@ How do I support multiple languages of documentation?
 
 See the section on :ref:`Localization of Documentation`.
 
-Do I need to be whitelisted?
```
Comment (humitos) [inline docs/faq.rst]: Why did you remove this one?

Comment (ericholscher) [inline docs/faq.rst]: It's years old, and talks about something that hasn't existed for a long time.
